packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Asset/index.tsx

```diff
@@ -422,15 +404,40 @@ const NftAsset: React.FC<Props> = ({
                       />
                       <div style={{ display: 'flex', justifyContent: 'center' }}>
                         <Socials>
+                          <SocialLink>
+                            <BlockchainIcon
+                              onClick={() => {
+                                analyticsActions.trackEvent({
+                                  key: Analytics.NFT_REFRESH_METADATA_CLICKED,
+                                  properties: {}
+                                })
+                                // refreshMetadata()
```

Comment:
does this need to be here?

Reply:
yeah, need to add that function as part of another ticket

Reply:
@rsteubs-bc lets remove refreshMetadata